css-four-column-layout/styles.css

```diff
@@ -0,0 +1,55 @@
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  background-color: whitesmoke;
+  margin: 50px auto;
+  padding: 50px 25px;
+  border-bottom: 50px;
+}
+
+h2 {
+  display: flex;
+  justify-content: center;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+}
+
+.col-one-fourth {
```

Comment:
since the column class is designed to set the width of a column, you want to set the width here, not the height. and since it is 1/4, that width should be 25%